plugins/package-managers/gradle/src/main/resources/init.gradle

```diff
@@ -409,7 +409,7 @@ class AbstractOrtDependencyTreePlugin<T> implements Plugin<T> {
             // available since Gradle 6.0.
             boolean isDeprecatedForResolving = false
             if (configuration.metaClass.respondsTo(configuration, 'getResolutionAlternatives')) {
-                isDeprecatedForResolving = configuration.getResolutionAlternatives() != null
+                isDeprecatedForResolving = !configuration.getResolutionAlternatives().isEmpty()
```

Comment:
For "Groovy Truth" you *must* drop the `.isEmpty()` here.

Reply:
Okay, was not the `isEmpty`() but the negation operator that caused the problem.

Reply:
Yeah, sorry, that was wrong in my proposal.